specs/config/config-resources.html

```diff
@@ -261,7 +304,22 @@ <h4>Configuration context</h4>
           >
         </li>
       </ul>
-      <p>TBD - define syntax of Configuration-Context header by referring to IETF RFC.</p>
+      <div class="conformance">
```

Comment:
I thought you are making just a few more edits on top of the existing PR? Why are these not in the master branch yet?